x-pack/solutions/observability/plugins/observability/server/features/cases_v1.ts

```diff
@@ -10,7 +10,7 @@ import { hiddenTypes as filesSavedObjectTypes } from '@kbn/files-plugin/server/s
 import { i18n } from '@kbn/i18n';
 import { KibanaFeatureConfig, KibanaFeatureScope } from '@kbn/features-plugin/common';
 import { CasesUiCapabilities, CasesApiTags } from '@kbn/cases-plugin/common';
-import { casesFeatureId, casesFeatureIdV2, observabilityFeatureId } from '../../common';
+import { casesFeatureId, casesFeatureIdV3, observabilityFeatureId } from '../../common';
 
 export const getCasesFeature = (
```

Comment:
Why does the cases feature exist in the observability plugin to begin with? I'm simply not familiar with the history.